timers/timers.go

```diff
@@ -95,10 +95,8 @@ func (e *Timers) setTimeout(callback goja.Callable, delay float64, args ...goja.
 	go func() {
 		timer := time.NewTimer(time.Duration(delay * float64(time.Millisecond)))
 		defer func() {
+			timer.Stop() // apparently there is no need to drain the channel, and it sometimes deadlocks :facepalm:
```

Comment:
```suggestion
			timer.Stop()
```
Do we really need it?